Description: Avoid test failures due to i18n and of expected output
 Force such tests to run with English messages.

 Patch backported from upstream git.
Author: Olly Betts <olly@survex.com>

---
Last-Update: 2012-02-28

diff --git a/tests/cavern.tst b/tests/cavern.tst
index 65ea6ed..7a02132 100755
--- a/tests/cavern.tst
+++ b/tests/cavern.tst
@@ -59,7 +59,7 @@ test -x "$testdir"/../src/cavern || testdir=.
 
 LC_ALL=C
 export LC_ALL
-unset SURVEXLANG
+SURVEXLANG=en
 export SURVEXLANG
 
 for file in $TESTS ; do
@@ -268,9 +268,9 @@ for file in $TESTS ; do
     if test -f "$file.out" ; then
       # Check output is as expected.
       if test -n "$VERBOSE" ; then
-	diff tmp.out "$file.out" || exit 1
+	sed '1,/^$/d' tmp.out|diff - "$file.out" || exit 1
       else
-	diff tmp.out "$file.out" > /dev/null || exit 1
+	sed '1,/^$/d' tmp.out|diff - "$file.out" > /dev/null || exit 1
       fi
     fi
     rm -f tmp.*
diff --git a/tests/diffpos.tst b/tests/diffpos.tst
index d36fa50..cffde1a 100755
--- a/tests/diffpos.tst
+++ b/tests/diffpos.tst
@@ -1,7 +1,7 @@
 #!/bin/sh
 #
 # Survex test suite - diffpos tests
-# Copyright (C) 1999-2003,2010 Olly Betts
+# Copyright (C) 1999-2003,2010,2012 Olly Betts
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
@@ -31,6 +31,9 @@ test -x "$testdir"/../src/cavern || testdir=.
 
 : ${TESTS=${*:-"delatend addatend"}}
 
+SURVEXLANG=en
+export SURVEXLANG
+
 for file in $TESTS ; do
   echo $file
   rm -f diffpos.tmp
diff --git a/tests/equatenosuchstn.out b/tests/equatenosuchstn.out
index 39de2d5..76a4d16 100644
--- a/tests/equatenosuchstn.out
+++ b/tests/equatenosuchstn.out
@@ -1,6 +1,3 @@
-Survex 1.2.5
-Copyright (C) 1990-2012 Olly Betts
-
 Removing trailing traverses...
 
 Concatenating traverses between nodes...
